msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -67,14 +67,14 @@ public PublicClientApplication(string clientId, string authority)
 
 #if WINRT
 /// <summary>
-/// 
+///
```

Comment:
should add comment. @JM can help if needed to provide a good string that developers will actually understand

Reply:
@henrik-me @rayluo 
I was thinking of doing a pass to propose XML comment improvements on all the MSAL.NET public API.

Reply:
Quoted from a comment I wrote via github: "Let's create a new PR for that". That PR is now created. Let's follow up with any further conversation on this topic in that PR. [https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1165](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1165)

---
In reply to: [205348063](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1118#discussion_r205348063) [](ancestors = 205348063)